### ISSUE TITLE
How can I make call from one android device to another device

### ISSUE THREAD

**Issue by krishnameena** [danInAustralia/WalkieTalkie#5]
I have imported it to android studio and when I am starting the app , I can see only blank screen In Range Peers is shown

**Comment by danInAustralia** [comment]
Hi,

if you install the app on two devices, they should be able to see each other - from there you should be able to establish a physical (wifi-direct) connection between the two. I will re-establish the transmission of audio soon.

**Comment by krishnameena** [comment]
Yes I am able to establish a physical connection but I am waiting for audio transmission.

**Comment by Psarmmiey** [comment]
Please how do I establish direct wifi connection between the two devices and what about viewing peers on an already established network thereby selecting a user to call.

**Comment by danInAustralia** [comment]
Hi @Psarmmiey, thanks for the message.

This project was initially made as a proof of concept, it probably doesn't run seamlessly on all devices. I also haven't been able to dedicate much time to this due to other projects. I wish to look at it again soon.

To establish wifi connection between the two devices, first start the app on both devices. You can 'make a call' from one device by choosing another device in the list of peers. The other device, should then prompt the user that someone is trying to make a call. The call will be established with the 'accept' button is pressed.

**Comment by danInAustralia** [comment]
Hi all, try again. I have reinstated the full-duplex audio and call receiver in this proof of concept. It's still rough around the edges, but you should be able to see other devices that are already running the app. You can choose one to connect to and hear the mic audio from each device in the respective device. Wear headphones or keep some distance between devices or you will get audio feedback.

**Comment by ibrahim-sdsol** [comment]
Hi @danInAustralia  can you please guide, how can i use the multicast socket to enable group chat. 
is it a good idea to connect using multi cast sockets  for a group audio call?
